core/src/main/java/hudson/model/Slave.java

```diff
@@ -133,10 +133,7 @@
     private /*almost final*/ DescribableList<NodeProperty<?>,NodePropertyDescriptor> nodeProperties = 
                                     new DescribableList<NodeProperty<?>,NodePropertyDescriptor>(Jenkins.getInstance().getNodesObject());
 
-    /**
-     * Lazily computed set of labels from {@link #label}.
-     */
-    private transient volatile Set<Label> labels;
```

Comment:
@rpau and now image that not all plugins under jenkinsci github account are hosted. Any clean-up change adds probability in compatibility breakage that jenkins is keeping so many years. You can propose such step for 2.0 as it will expect breakages. Removing unused imports imho relates to coding style (thread that i started in dev list).